README.md

```diff
@@ -88,6 +88,8 @@ cisco_nxapi (1.0.0)
 net_http_unix (0.2.1)
 ~~~
 
+* Please note: The `ciscopuppet` module requires a compatible `cisco_node_utils` gem. This is not an issue with release versions; however, when using a pre-release module it may be necessary to manually build a compatible gem. Please see the `cisco_node_utils` developer's guide for more information on building a `cisco_node_utils` gem:  [README-develop-node-utils-APIs.md](https://github.com/cisco/cisco-network-node-utils/blob/develop/docs/README-develop-node-utils-APIs.md#step-5-build-and-install-the-gem) *
```

Comment:
It'll be a broken link then. This is a NU URL that I'm adding to our Puppet doc.